UAGimbalAdapter.m

```diff
@@ -45,24 +61,10 @@ - (void)dealloc {
     self.placeManager.delegate = nil;
 }
 
-
-+ (instancetype)shared {
-    static dispatch_once_t onceToken = 0;
-    dispatch_once(&onceToken, ^{
-        _sharedObject = [[self alloc] init];
-    });
-
-    return _sharedObject;
-}
-
-+ (void)handleAppDidFinishLaunching {
-    [[NSNotificationCenter defaultCenter] removeObserver:[UAGimbalAdapter class]
-                                                    name:UIApplicationDidFinishLaunchingNotification
-                                                  object:nil];
-
-    if ([GMBLPlaceManager isMonitoring]) {
-        [[UAGimbalAdapter shared] startWithGimbalAPIKey:nil];
-    }
+- (void)restore {
+    self.started = [Gimbal isStarted];
```

Comment:
If all we want from this function is re-initialize after the app was killed then this works. No need to check isStarted before setting the delegate or setting the deviceAttributes that I'm aware of.